HerPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -156,7 +156,10 @@ public EpcRating DisplayEpcRating
 
     public bool EpcIsTooHigh => EffectiveEpcBand is EpcRating.A or EpcRating.B or EpcRating.C;
 
-    public bool IncomeIsTooHigh => IncomeBand is (IncomeBandEnum.GreaterThan31000 or IncomeBandEnum.GreaterThan34500) && IsLsoaProperty is not true;
+    #pragma warning disable CS0618
+    public bool IncomeIsTooHigh => IncomeBand is (IncomeBandEnum.GreaterThan31000 or IncomeBandEnum.GreaterThan34500 or IncomeBandEnum.GreaterThan36000) && IsLsoaProperty is not true; //Obsolete Income Bands used to preserve backwards-compatibility

```

Comment:
Are we covering this with any tests?

Reply:
I've left  a note on the comment above where I think these are covered.